app/src/main/res/layout/activity_await_help.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:id="@+id/await_help_layout"
+    tools:context=".AwaitHelpActivity"
+    android:orientation="horizontal">
+
+    <TextView
+        android:id="@+id/textView3"
```

Comment:
I would change this id to a more meaningful one to avoid having conflicts with the others.

Reply:
You're right, this was just a temporary textView until we actually implement this page, but it's always better to be thorough!